tests/unit/src/main/java/org/mule/tck/junit4/AbstractMuleContextTestCase.java

```diff
@@ -127,7 +127,7 @@
   /**
    * Default timeout used when blocking on {@link org.reactivestreams.Publisher} completion.
    */
-  public static final int BLOCK_TIMEOUT = 200;
+  public static final int BLOCK_TIMEOUT = 500;
```

Comment:
This shouldn't need to be so high, at least not where I've used it most. Do we know 500ms is needed on build server?